tests/func/test_get.py

```diff
@@ -87,3 +88,18 @@ def test_get_to_dir(dname, erepo):
 
     assert os.path.isdir(dname)
     assert filecmp.cmp(erepo.FOO, dst, shallow=False)
+
+
+def test_get_from_non_dvc_master(empty_dir, erepo, caplog):
+    erepo.dvc.scm.repo.index.remove([".dvc"], r=True)
```

Comment:
Oh, so you don't even need to gitignore anything 🙂So that erepo change was not needed at all? 🙂

Reply:
Yep, that's true. :)